.github/workflows/checkPot.yml

```diff
@@ -0,0 +1,42 @@
+name: Check comments for translators
+
+on:
+  push:
+    branches:
+    - master
+    - beta
+    - rc
+    - try-**
+    - release-**
+    - githubActions # To be removed
+
+  pull_request:
+    branches:
+    - master
+    - beta
+    - rc
+    - try-**
+    - release-**
+
+  workflow_dispatch:
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
+env:
+  if: $GITHUB_EVENT_NAME == 'pull_request'
+  pullRequestNumber: ${{ github.event.number }}
+jobs:
+  checkPot:
+    runs-on: windows-latest
+    defaults:
+      run:
+        shell: cmd
+    steps:
+    - name: Checkout NVDA
+      uses: actions/checkout@v4
+      with:
+        submodules: true
+    - name: Build NVDA
+      uses: ./nvdaBuilder
+    - name: Check comments for translators
+      run: powershell.exe ci/scripts/tests/translationCheck.ps1
```

Comment:
Can we build and check pot without a full build and potentially move this action to linux?